File: packages/expo-font/android/src/main/java/expo/modules/font/exceptions/FontFileInvalidException.java
package expo.modules.font.exceptions;

import org.unimodules.core.errors.CodedRuntimeException;

public class FontFileInvalidException extends CodedRuntimeException {
  public FontFileInvalidException(String path) {
    super(String.format("File '%s' isn't a valid font file.", path));
  }

  @Override
  public String getCode() {
    return "ERR_FONT_FILE_INVALID";
  }
}

File: packages/expo-font/android/src/main/java/expo/modules/font/exceptions/FontFileInvalidException.kt
package expo.modules.font.exceptions

import org.unimodules.core.errors.CodedRuntimeException

class FontFileInvalidException(path: String?) : CodedRuntimeException(String.format("File '%s' isn't a valid font file.", path)) {
  override fun getCode(): String {
    return "ERR_FONT_FILE_INVALID"
  }
}